package/boot/uboot-sunxi/patches/253-sunxi-h5-add-support-for-nanopi-r1s.patch

```diff
@@ -0,0 +1,295 @@
+--- /dev/null
++++ b/arch/arm/dts/sun50i-h5-nanopi-r1s-h5.dts
+@@ -0,0 +1,265 @@
++// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
++/*
++ * Copyright (C) 2021 Chukun Pan <amadeus@jmu.edu.cn>
++ *
++ * Based on sun50i-h5-nanopi-neo-plus2.dts, which is:
++ *   Copyright (C) 2017 Antony Antony <antony@phenome.org>
++ *   Copyright (C) 2016 ARM Ltd.
++ */
++
++/dts-v1/;
++#include "sun50i-h5.dtsi"
++
++#include <dt-bindings/gpio/gpio.h>
++#include <dt-bindings/input/input.h>
++
++/ {
++	model = "FriendlyARM NanoPi R1S H5";
```

Comment:
Did you send this also to upstream U-boot to get it integrated there?

Reply:
no, I is mandatory for this MR to get merged? as we donlt use latest in openwrt I thought it will get picked up later one when U-boot will sync with upstream linux as I notice they do about every year

Reply:
Should be ok, as it is already in upstream kernel.

Reply:
alright. I'll send this also to u-boot upstream. They don't use github but mail patch only so a little challenging for me :) 
Anyway I hope we can move along wit this MR and not wait for u-boot upstream.